### ISSUE TITLE
SVG text paint-order = "stroke" is not supported

### ISSUE THREAD

**Issue by aziesemer** [Kozea/CairoSVG#218]
If you try this code:

<svg id="floor2" class="floor" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="-33250 -7500 800 250">
<style type="text/css">.ruler-text.metric{fill:#000; font:107pt arial; text-anchor:middle; alignment-baseline:middle; paint-order: stroke; stroke: #ff0; stroke-width: 10pt; stroke-linecap: butt; stroke-linejoin: miter}</style>
<text transform="translate(-32852.4 -7353.87) rotate(0)" class="ruler-text metric">2.66m</text>
</svg>

You'll end up with:
<img width="830" alt="screen shot 2018-11-15 at 9 54 34 am" src="https://user-images.githubusercontent.com/11475314/48560772-9cfb2300-e8bc-11e8-8041-809b5b0a7acf.png">
instead of:
<img width="733" alt="screen shot 2018-11-15 at 9 55 51 am" src="https://user-images.githubusercontent.com/11475314/48560797-aab0a880-e8bc-11e8-8c2d-96ae9089e96b.png">

PR #217 fixes that.

**Comment by liZe** [comment]
`paint-order` has been added in SVG 2.0 and CairoSVG doesn't have a clear plan to support this new version :smile:. By the way, this change seems harmless and there's no reason not to include it. Thank you!

**Comment by liZe** [comment]
Oh, I didn't realize that #217 and #218 were conflicting. I have to check this.

**Comment by aziesemer** [comment]
@liZe Yes, I have it merged in my computer but had to slit in order to produce 2 different PRs.

**Comment by aziesemer** [comment]
@liZe I fixed the conflict in the same PR and it should work now.
